src/content/blog/whats-new-october.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: "What's new in Astro - October"
+description: "Another busy month in Astro! From Hacktoberfest to releasing Page Partials. Read up on everything that happened this month in the Astro universe."
+publishDate: "October 31, 2023"
+authors:
+  - elian
+coverImage: "/src/content/blog/_images/astro-3/blog-hero-post-header.webp"
+socialImage: "/src/content/blog/_images/astro-3/blog-social.webp"
+lang: "en"
+---
+
+## Astro releases
+
+- Astro `3.2`
+  - Control the browser history stack
+  - Trigger page navigation via JavaScript
+  - Route announcer for screen readers
+  - Dynamically add integrations
+  - [Read more about it here](/blog/astro-320/)
+- Astro `3.3`
+  - experimental `<Picture />` component
+  - Syntax highlighting improvements
+  - Package Provenance
+  - Quality of Life improvements
+  - [read more about it here](/blog/astro-330/)
+- Astro `3.4`
+  - Page Partials
+  - Image optimisation performance
+  - Experimental dev overlay
+  - [read more about it here](/blog/astro-340/)
+
+## Hacktoberfest
+
+[Hacktober](https://hacktoberfest.com/) is officially over now! Thank you all for your amazing contributions and PRs! Here are some of our statistics:
+
+- 2 Astro core PRs
+- 49 Starlight PRs
+- 62 Docs PRs
+
+## Docs
+
+We extended the tutorial with some requested features!
+
+- [Content Collections](https://docs.astro.build/en/tutorials/add-content-collections/)
+- [View Transitions](https://docs.astro.build/en/tutorials/add-view-transitions/)
+
+The Astro Docs team is also working on a "Rendering Modes" page, it's still in the works and we'd love your feedback on [the open PR](https://github.com/withastro/docs/pull/4667)
+
+## Starlight
+
+### Releases & new features
```

Comment:
Again, not sure whether this feels like a lot here. Maybe something short and snazzier?

Reply:
Follow up, suggest NOT including it this month in favour of the statement above at the end of releases!